src/abstract_block.jl

```diff
@@ -54,7 +54,7 @@ end
 
 Return an iterator of occupied locations of `x`.
```

Comment:
```suggestion
Return a tuple of occupied locations of `x`.
```

Reply:
Nice catch, now I have updated the docstring.